common/workunit/pkgimpl.hpp

```diff
@@ -57,7 +57,7 @@ class CPackageSuperFileArray : implements ISimpleSuperFileEnquiry, public CInter
     {
         ForEachItemIn(idx, subFiles)
         {
-            if (stricmp(subFiles.item(idx).queryProp("@value"), subname))
+            if (strieq(subFiles.item(idx).queryProp("@value"), subname))
```

Comment:
What would the effect of this be? @afishbeck ?

Reply:
Looks like this is probably never called from here... it's emulating functionality that exists in the dali interface.  If it were called it wouldn't have worked, and this would be a fix.

Reply:
@richardkchapman